### ISSUE TITLE
how many iteration to get a resonable result

### ISSUE THREAD

**Issue by LovPe** [CharlesShang/FastMaskRCNN#146]
i followed this project and make some changes:
1/ use different conv layers for FPN
2/ use 1 scales for [2, 3, 4] feature pyramid layers and 3 scales for[5] layer
3/ use adam instead of monent SGD
and after runing ~25k iters, i get result like this:

![2017-09-13 21-37-48](https://user-images.githubusercontent.com/11580882/30382489-d371105c-98d1-11e7-863a-68324efeb4b7.png)
![2017-09-13 21-39-53](https://user-images.githubusercontent.com/11580882/30382490-d3bb444c-98d1-11e7-8c6d-768fe663f960.png)
![2017-09-13 21-40-38](https://user-images.githubusercontent.com/11580882/30382511-e237798c-98d1-11e7-8897-6b9026322bd8.png)
![2017-09-13 21-50-31](https://user-images.githubusercontent.com/11580882/30382112-d9fe25be-98d0-11e7-88e6-917b3280a7e6.png)
![2017-09-13 21-51-42](https://user-images.githubusercontent.com/11580882/30382121-e0143fc4-98d0-11e7-93fd-efd95b9ad5cb.png)
![2017-09-13 21-53-13](https://user-images.githubusercontent.com/11580882/30382131-e552ed8c-98d0-11e7-9717-dac87e5c63b3.png)
![2017-09-13 21-54-10](https://user-images.githubusercontent.com/11580882/30382140-eb0ede34-98d0-11e7-8503-b5090590ffc1.png)
![2017-09-13 21-54-45](https://user-images.githubusercontent.com/11580882/30382146-f2095eb2-98d0-11e7-9098-aee95b8a236d.png)

**here is the problem**
1/ it seems that there are many image with person, so may this class dominate the training and let the net converge to a sub opeimal , and how to overcome this problem?
2/ i tried  diffetent proportion of 5 loss[rpn_box, rpn_cls, rcnn_box, rcnn_cls, mask] and cant find some rules . do any one have idea how to choose this proportion
3/ do anyone get result as good as the paper present?

**Comment by shamanez** [comment]
@LovPe  how did you train this ?

**Comment by shamanez** [comment]
Specifications of the machine

**Comment by LovPe** [comment]
@shamanez  GTX 1080TI 11GB

**Comment by MansourTrabelsi** [comment]
@LovPe after the training, how did you do the test on an image?

**Comment by TMTING** [comment]
I also want to know how to test on my image,please help me,thanks.

**Comment by LovPe** [comment]
@TMTING i'm sorry that i already give up following this project, you can find my latest project here:
[https://github.com/LovPe/tf_rcnn]; 
In this project, I re-implement the orignal project but i think it may help . 
The test code is in the inference folder. Good luck
By the way, I strongly commend you to try a Keras Implement here: 
[github.com/matterport/Mask_RCNN]
